cicecore/cicedyn/infrastructure/io/io_netcdf/ice_history_write.F90

```diff
@@ -137,654 +138,653 @@ subroutine ice_write_hist (ns)
-                  'angle grid makes with latitude line on U grid', &
-                  'radians')
-      var_grd(n_ANGLE)%coordinates = 'ULON ULAT'
-      var_grd(n_ANGLET)%req = coord_attributes('ANGLET', &
-                  'angle grid makes with latitude line on T grid', &
-                  'radians')
-      var_grd(n_ANGLET)%coordinates = 'TLON TLAT'
-
-      ! These fields are required for CF compliance
-      ! dimensions (nx,ny,nverts)
-      var_nverts(n_lont_bnds) = coord_attributes('lont_bounds', &
-                  'longitude boundaries of T cells', 'degrees_east')
-      var_nverts(n_latt_bnds) = coord_attributes('latt_bounds', &
-                  'latitude boundaries of T cells', 'degrees_north')
-      var_nverts(n_lonu_bnds) = coord_attributes('lonu_bounds', &
-                  'longitude boundaries of U cells', 'degrees_east')
-      var_nverts(n_latu_bnds) = coord_attributes('latu_bounds', &
-                  'latitude boundaries of U cells', 'degrees_north')
-      var_nverts(n_lonn_bnds) = coord_attributes('lonn_bounds', &
-                  'longitude boundaries of N cells', 'degrees_east')
-      var_nverts(n_latn_bnds) = coord_attributes('latn_bounds', &
-                  'latitude boundaries of N cells', 'degrees_north')
-      var_nverts(n_lone_bnds) = coord_attributes('lone_bounds', &
-                  'longitude boundaries of E cells', 'degrees_east')
-      var_nverts(n_late_bnds) = coord_attributes('late_bounds', &
-                  'latitude boundaries of E cells', 'degrees_north')
+         !-----------------------------------------------------------------
+         ! Define attributes for time bounds if hist_avg is true
+         !-----------------------------------------------------------------
+
+         if (hist_avg(ns) .and. .not. write_ic) then
+            dimid(1) = boundid
+            dimid(2) = timid
+            status = nf90_def_var(ncid,'time_bounds',lprecision,dimid(1:2),varid)
+            call ice_check_nc(status, subname// ' ERROR: defining var time_bounds', &
+                              file=__FILE__, line=__LINE__)
+            status = nf90_put_att(ncid,varid,'long_name', 'time interval endpoints')
+            call ice_check_nc(status, subname// ' ERROR: time_bounds long_name', &
+                              file=__FILE__, line=__LINE__)
+            write(cdate,'(i8.8)') idate0
+            write(title,'(a,a4,a1,a2,a1,a2,a1,i2.2,a1,i2.2,a1,i2.2)') 'days since ', &
+                  cdate(1:4),'-',cdate(5:6),'-',cdate(7:8),' ', &
+                  hh_init,':',mm_init,':',ss_init
+            status = nf90_put_att(ncid,varid,'units',title)
+            call ice_check_nc(status, subname// ' ERROR: time_bounds units', &
+                              file=__FILE__, line=__LINE__)
+            if (days_per_year == 360) then
+               status = nf90_put_att(ncid,varid,'calendar','360_day')
+               call ice_check_nc(status, subname// ' ERROR: time calendar1', &
```

Comment:
As above, here you're calling them calendar1, 2 and 3. So perhaps make them all align w/ the calendar you're retrieving?